input/fsh/at-core-patient.fsh

```diff
@@ -1,14 +1,15 @@
 /*##############################################################################
 # Type:       FSH-File for an FHIR® Profile
 # About:      HL7® Austria FHIR® Core Profile for Patient.
-# Created by: HL7® Austria, TC FHIR® 
+# Created by: HL7® Austria, TC FHIR®
 ##############################################################################*/
 
 Profile:        HL7ATCorePatient
 Parent:         Patient
 Id:             at-core-patient
 Title:          "HL7® AT Core Patient Profile"
-Description:    "HL7® Austria FHIR® Core Profile for patient pata in Austria."
+Description:    "HL7® Austria FHIR® Core Profile for patient data in Austria.
+The HL7 AT Core Patient is based upon the core FHIR Patient Resource and designed to meet the applicable patient demographic data elements in Austria. It identifies which core elements, extensions, vocabularies and value sets SHALL be present in the resource when using this profile."

```

Comment:
Maybe add this sentence: "Note, this extension represents the common structure of address information within Austrian information systems."

Also -> FHIR and HL7 are both missing ®

Reply:
Sentence and ® have been added.